src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/wiremock/NonAssociationsApiMockServer.kt

```diff
@@ -147,6 +148,9 @@ class NonAssociationsApiMockServer : WireMockServer(WIREMOCK_PORT) {
 
   fun createNonAssociationSynchronisationCount() =
     findAll(putRequestedFor(urlMatching("/sync/upsert"))).count()
+
+  fun createNonAssociationMigrationCount() =
+    findAll(postRequestedFor(urlMatching("/migrate"))).count()
```

Comment:
I would have expected this to include the `/non-associations` path too?

Reply:
That doesn't work - it is only matching on the /migrate